documentation/reference/document-v1-api-reference.html

```diff
@@ -273,18 +273,27 @@ <h2 id="request-body">Request body</h2>
 
 
 <h2 id="http-status-codes">HTTP status codes</h2>
-Non-exhaustive list of status codes:
+<p>
+  See <a href="https://github.com/vespa-engine/vespa/issues/13465">13465</a>
+  for defined document types not assigned to a content cluster.
+</p><p>
+  Non-exhaustive list of status codes:
+</p>
 <table class="table">
 <tr>
   <th>200</th>
   <td>OK. Note that updating a non-existing document is defined as success.</td>
+</tr><tr>
+  <th>400</th>
+  <td>Bad request. Returned for undefined document types.
```

Comment:
It's returned to various input errors, not just document types without a cluster.

Reply:
reformulated